DictionaryTags.txt

```diff
@@ -636,34 +636,12 @@ alias_tag(proj_item_ch_name_list) sparse_ch_name_list
 xplotter_layout         3501	string      -	"Xplotter layout tag"
 
 #==============================================================================================================
-#               3502...
-#               3799          Reserved for MNE estimates (MHa)
+#		3502...
+#		3999          Reserved for MNE estimates (MHa)
 #
 # This range is reserved for MGH and Matti Hamalainen for developing
-# MNE software.
+# MNE software. See DictionaryTags_MNE.txt.
 #
-# THIS TAG RANGE IS POTENTIALLY CONFIDENTIAL INFORMATION.
-# DO NOT PUBLISH WITHOUT PERMISSION.
-#==============================================================================================================
-#<reserved>  		3502    -
-#...
-#<reserved>		3799    -
-#==============================================================================================================
-:id mne
-#==============================================================================================================
-ch_name_list            3507	string      -	"List of channel names"
-
-
-#==============================================================================================================
-#
-#		3800...
-#		3999	    Range of free tags
-#
-#<free>  		3800    -
-#...
-#<free>			3999    -
-#
-#==============================================================================================================
```

Comment:
Removed all of this because it's clearer just to point people to the companion file I think.

Reply:
Sorry, maybe I did not quite get what was the change. I suppose these lines were just a placeholder to show a free tag range?

Reply:
There is the `ch_name_list            3507` farther up, but this is redundant with the [definition in the _MNE.txt](https://github.com/mne-tools/fiff-constants/blob/master/DictionaryTags_MNE.txt#L39)